Clients/src/presentation/components/FileUpload/index.tsx

```diff
@@ -0,0 +1,170 @@
+import React from "react";
+import { Stack, Typography, Button } from "@mui/material";
+import Uppy from "@uppy/core";
+import { DragDrop } from "@uppy/react";
+import "@uppy/core/dist/style.css";
+import "@uppy/drag-drop/dist/style.css";
+import UploadSmallIcon from "../../assets/icons/file-upload.svg"; 
+
+const FileUploadComponent: React.FC = () => {
+  const uppy = new Uppy({
+    restrictions: {
+      maxNumberOfFiles: 1,
+      allowedFileTypes: [".pdf"],
+      maxFileSize: 50 * 1024 * 1024,
+    },
+    autoProceed: false,
+  });
```

Comment:
@salmanja 
This one is also very important and good stuff.
This will help note to mess with browsers functionality.
would you please check it out and apply? 🙂

Reply:
> @salmanja 
> 
> This one is also very important and good stuff.
> 
> This will help note to mess with browsers functionality.
> 
> would you please check it out and apply? 🙂

Of course!